packages/manager/apps/dedicated/client/app/network-security/network-security.controller.js

```diff
@@ -0,0 +1,5 @@
+export default class NetworkSecurityCtrl {
+  onScrubbingCenterTabClick() {
+    return this.goToScrubbingCenter();
+  }
+}
```

Comment:
I think no need.
You can use directly goToScrubbingCenter() in you template. Without this controller.